app/assets/stylesheets/application.css.scss

```diff
@@ -24,3 +24,18 @@ input {
   border-radius: 5px;
   margin-bottom: 1rem;
 }
+
+.field input,
+.actions input[type="submit"] {
+  width: 100%;
+}
+
+.field input [type="checkbox"] {
+  width: auto;
+}
+
+.remember {
+  display: flex;
+  align-items: baseline;
+  gap: 8px;
+}
```

Comment:
- Kindly make sure your design match that of the wireframe
Your Design | Wireframe
![blog_login](https://user-images.githubusercontent.com/44624138/180497626-c5aaf221-0de6-4fbb-8ca3-815cf08935bc.png) | ![Screenshot from 2022-07-22 18-59-31](https://user-images.githubusercontent.com/44624138/180497671-8f7f217d-5848-4e5f-8659-926f9b76aae6.png)